app/Jobs/ImportTurboVotePosts.php

```diff
@@ -153,6 +153,22 @@ public function handle(Rogue $rogue)
         ]);
     }
 
+    /**
+     * Initiate the stat counters to use with imports to Rogue.
+     *
+     * @return array
+     */
+    public function statsInit()
+    {
+        return [
+            'totalRecords' => 0,
+            'countScrubbed' => 0,
+            'countProcessed' => 0,
+            'countPostCreated' => 0,
+            'countUserAccountsCreated' => 0,
+        ];
+    }
+
     /**
```

Comment:
I think I am confused! Why wouldn't this work to keep in the `ImportToRogue` trait?

Reply:
@chloealee I left it in there too. I am just hesitant to make any changes to this file because there are no tests so we'd have to test manually and I'm not sure what I'm looking for in the results. Does that seem reasonable or nah?

Reply:
oh right right ! That totally makes sense! Sorry - was thinking this was for Rock the Vote!

Reply:
ah no worries! I know a lot of work and testing and analysis and planning went into the TurboVote import so I have an "if it ain't broke don't fix it" view on it currently

Reply:
agreed!

Reply:
@DFurnes done and done!